temporian/core/operators/simple_moving_average.py

```diff
@@ -64,6 +66,9 @@ def __init__(
 
         self.check()
 
+    def window_length(self):
+        return self._window_length
+
     @classmethod
     def build_op_definition(cls) -> pb.OperatorDef:
         return pb.OperatorDef(
```

Comment:
Should the `window_length` attribute be a float now?

`type=pb.OperatorDef.Attribute.Type.STRING,`

Reply:
Good catch.
*Done